### PR TITLE
3 stagev4

### PR DESCRIPTION
**Pull request by mkestra**
added functionality for 3 independent dumb motor stages with limit switches on RTDSK0.

Includes a configuration change to 3 x EL7041 terminals on the RTDSK0 Dinn rail, replacing 3 x EL7041-1000 terminals. Also added EL1004 to ST1K4 for @nrwslac,

Finally, added one line to hard-code a semi-permanent override to the lower limit switch on south motor of SL1K0 until it can be further investigated in the winter down.

### CODE REVIEW

**Review by ghalym — APPROVED** [review]

**Review by ZLLentz — CHANGES_REQUESTED** [review]
Can you update the PR description to reflect the changes made in the PR? It will be annoying to go back to this later and try to understand what was changed without any relevant PR text.

**Comment by mkestra** [comment]
changed description to describe changes in more detail.

File: plc-kfe-motion/kfe_motion/POUs/PRG_RTDSK0.TcPOU [hunk 1]
```diff
@@ -98,7 +114,39 @@ RTDSK0_MPA_04(
     i_xReset := ,
     io_fbFFHWO := GVL.fbFastFaultOutput1,
 );
-*)]]></ST>
+*)
+
+//initialize RTDSK0 stages
+    IF ( bInit) THEN
```
Comment (ZLLentz) [inline plc-kfe-motion/kfe_motion/POUs/PRG_RTDSK0.TcPOU:120]: This isn't important but it triggers my OCD again
```suggestion
IF bInit THEN
```

File: plc-kfe-motion/kfe_motion/POUs/PRG_RTDSK0.TcPOU [hunk 2]
```diff
@@ -98,7 +114,39 @@ RTDSK0_MPA_04(
...
+    Main.M12.nHomingMode := ENUM_EpicsHomeCmd.HIGH_LIMIT;
+    Main.M13.nHomingMode := ENUM_EpicsHomeCmd.HIGH_LIMIT;
+    
+    bInit := FALSE;
+END_IF
+    
+    fbRTDSK0_SCRP_MMS_Z (stMotionStage := Main.M11);
+    fbRTDSK0_SCRP_MMS_X (stMotionStage := Main.M12);
+    fbRTDSK0_SCRP_MMS_Y (stMotionStage := Main.M13);
+    FFO(i_xOK := NOT(Main.M13.bLimitForwardEnable) AND Main.M13.bLimitBackwardEnable, io_fbFFHWO := GVL.fbFastFaultOutput1);]]></ST>
```
Comment (ZLLentz) [inline plc-kfe-motion/kfe_motion/POUs/PRG_RTDSK0.TcPOU:149]: This looks correct (zxy mapping, FFO implementation), but why is this whole block randomly indented? Why are there random spaces between the function blocks and their parentheses? Why are there extra parentheses in the middle of the i_xOK boolean? It's not wrong it's just confusing.

**Review by ZLLentz — APPROVED** [review]
Functionality looks good to me, pointed out some nitpicks

**Pull request merged**